lib/cog_api/http/groups.ex

```diff
@@ -35,8 +40,8 @@ defmodule CogApi.HTTP.Groups do
   end
 
   defp update_membership(endpoint, group, user, action) do
-    path = "groups/#{group.id}/membership"
-    Base.post(endpoint, path, %{members: %{users: %{action =>  [user.username]}}})
+    path = "groups/#{group.id}/users"
+    Base.post(endpoint, path, %{users: %{action => [user.email_address]}})
```

Comment:
Would you mind updating the fake for groups so that it uses the email here? Currently it uses the username for the lookup.